WooCommerce/Classes/Extensions/Array+Helpers.swift

```diff
@@ -51,3 +51,16 @@ extension Sequence where Element: Numeric {
     /// Returns the sum of all elements in the collection.
     func sum() -> Element { return reduce(0, +) }
 }
+
+extension Sequence where Element: Equatable {
+    /// Returns the sequence with any duplicate elements after the first one removed.
+    func removingDuplicates() -> [Element] {
```

Comment:
nit (feel free to ignore it because I'm not 100% sure that it's the correct thing to do): what do you think of `removeDuplicates()`?

Reply:
good question! for this case, [Swift API design guidelines](https://swift.org/documentation/api-design-guidelines/) have a section "Name Mutating/nonmutating method pairs consistently" that recommends naming nonmutating function as the verb's past tense "ed" or "ing" whereas mutating functions have the verb in current tense. Lemme know if you have any thoughts!